en/0x10-V1-Inventory.md

```diff
@@ -17,13 +17,13 @@ transparency within the organization, promotes software standardization and reus
 
 | # | Description | L1 | L2 | L3 |
 | :---: | :--- | :---: | :---: | :---: |
-| **1.1** | All components and their versions are known at completion of a build | ✓ | ✓ | ✓ |
-| **1.2** | Package managers are used to manage all third-party binary components | ✓ | ✓ | ✓ |
```

Comment:
Cannot remove binary. There are issues with licensed components. Other orgs want to build from source code.

Reply:
This implies that you don't need a package manager if the component is not binary.